lib/SPIRV/libSPIRV/spirv.hpp

```diff
@@ -376,6 +376,8 @@ enum FPFastMathModeMask {
     FPFastMathModeNotInfMask = 0x00000002,
     FPFastMathModeNSZMask = 0x00000004,
     FPFastMathModeAllowRecipMask = 0x00000008,
+    FPFastMathModeAllowContractMask = 0x10000,
```

Comment:
I think it is better to add `INTEL` suffix to those to highlight that these do not belong to the code SPIR-V spec

Reply:
I also suggest to make the assigned value to be 0x00010000, so the changes would be:
```
    FPFastMathModeAllowContractFastINTELMask = 0x00010000,
    FPFastMathModeAllowReassocINTELMask = 0x00020000,
```